src/app/bav/fields.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     journeyKey: "sortCode",
     validate: [
       "required",
+      { type: "minlength", arguments: [6] },
```

Comment:
Does this cover digits or characters? Think the requirement was for digits

Reply:
I've added additional regexes to return the correct error messages in all scenarios, length and disallowed characters and combinations of the two

Reply:
Hmm do you want to catch up about this? So here, I have entered 4 digits but 6 characters - I should be seeing the "Your sort code must be 6 digits long" error message but instead I'm seeing this

![image](https://github.com/govuk-one-login/ipv-cri-bav-front/assets/40401118/3b03ffe1-5bfe-4524-812b-6a8b4684b379)